### PR TITLE
added step to presence troubleshooting

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1738

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: Teams/exchange-integration/teams-exchange-interaction-issue.md [hunk 1]
```diff
@@ -312,7 +312,18 @@ Try to access the REST API URL from a browser in the external network. If you ge
...
 
-#### Step 2: Verify that Teams isn't blocked from accessing EWS for the entire organization
+#### Step 2: Verify that the Teams Presence Based on Calendar Events test is successful
+
+1. Ask the user to go to the [Teams Presence Based on Calendar Events](https://testconnectivity.microsoft.com/tests/TeamsCalendarPresence/input) section of **Microsoft Remote Connectivity Analyzer**.
+
+:::image type="content" source="media/teams-exchange-interaction-issue/presence-test-RCA.png" alt-text="Screenshot of Teams Presence test":::
+
+2. Input the requested information.
+3. Select the Perform Test button to start the Teams Presence Based on Calendar Events test.
```
Comment (JohanFreelancer9) [inline Teams/exchange-integration/teams-exchange-interaction-issue.md:322]:
```suggestion
3. Select the **Perform Test** button to start the Teams Presence Based on Calendar Events test.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MaratMussabekov - kindly commit the minor suggested change. Many thanks.

**Comment by joaolloureiro** [comment]
@MaratMussabekov Changes look good. Thank for your help.

**Pull request merged**